integration-tests/jpa-postgresql/src/test/java/io/quarkus/it/jpa/postgresql/JPAReflectionInGraalITCase.java

```diff
@@ -20,4 +20,11 @@ public void testFieldAndGetterReflectionOnEntityFromServlet() throws Exception {
         RestAssured.when().get("/jpa/testreflection").then().body(is("OK"));
     }
 
+    // Since the driver can do some reflection magic we also want to make sure that the regular functionality works fine
+    // within an integration test.
+    @Test
+    public void testJPAFunctionalityFromServlet() throws Exception {
+        RestAssured.when().get("/jpa/testfunctionality").then().body(is("OK"));
+    }
+
```

Comment:
I believe this is unnecessary as we already do this in `JPAFunctionalityInGraalITCase`?